pkg/datastore/database.go

```diff
@@ -423,21 +268,14 @@ func (db *relationalDb) Drop(tableNames ...string) error {
 	return db.DropCascade(false, tableNames...)
 }
 
-func (db *relationalDb) DropCascade(cascade bool, tableNames ...string) (err error) {
-	// Initialize DB connection
-	if err = db.Initialize(); err != nil {
-		return err
-	}
-
-	if _, ok := os.LookupEnv(DB_ADMIN_USERNAME_ENV_VAR); !ok {
-		return ErrMissingEnvVar.WithValue(ENV_VAR, DB_ADMIN_USERNAME_ENV_VAR)
-	}
-
-	adminUsername := getAdminUsername()
-
+func (db *relationalDb) DropCascade(_ bool, tableNames ...string) (err error) {
```

Comment:
@krishnamiriyala, we should return the name to the first parameter (the boolean) and use it to distinguish the regular drop vs cascading drop

Reply:
nvm. We don't have DropCascade in the most recent version of the library. We don't really need it, since we are not creating relationships between DB tables